console/src/components/Services/Common/ComponentErrorBoundary.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component, ReactNode } from 'react';
```

Comment:
Let's call this file `ErrorBoundary`. `Component` is redundant as it is, in fact, a component.